qsiprep/cli/run.py

```diff
@@ -376,10 +376,11 @@ def get_parser():
         '--hmc_model', '--hmc-model',
         action='store',
         default='eddy',
-        choices=['none', '3dSHORE', 'eddy'],
+        choices=['none', '3dSHORE', 'eddy', 'hmcDTI'],
```

Comment:
what do you think of using just 'tensor' as the value instead of 'hmcDTI'?

Reply:
I'm fine with that.  I'll change it everywhere before my next push.